source/brailleDisplayDrivers/papenmeier_serial.py

```diff
@@ -182,11 +182,11 @@ def _handleKeyPresses(self): #called by the keycheck timer
 		"""if a button was pressed an input gesture is executed"""
 		if(self._dev!=None):
 			data = brl_poll(self._dev)
-			if(len(data) == 10 and data[1]=='K'):
-				pos = ord(data[2])*256+ord(data[3])
+			if len(data) == 10 and data[1] == ord('K'):
+				pos = data[2] * 256 + data[3]
 				pos = (pos-768)/3

```

Comment:
It looks like this can ever be negative. I'm not sure whether this is expected, but I don't think so. May be we should add a comment to this code that there are some code quality concerns here.